app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/ui/screens/codePager/CodePagerScreen.kt

```diff
@@ -199,31 +183,43 @@ fun CodeViewerFile(title: String, iconId: Int) {
 }
 
 @Composable
-@OptIn(ExperimentalMaterial3Api::class)
-private fun CodePagerTopAppBar(
-    scrollBehavior: TopAppBarScrollBehavior,
-    onBackPressed: () -> Unit,
-    title: String
+private fun CodePageView(
+    codePagerTitles: List<String>,
+    sampleData: Sample,
+    optionPosition: Int
 ) {
-    TopAppBar(
-        title = {
-            Text(
-                text = title,
-                color = MaterialTheme.colorScheme.onPrimary
-            )
-        },
-        colors = TopAppBarDefaults.topAppBarColors(
-            containerColor = MaterialTheme.colorScheme.primary,
-        ),
-        scrollBehavior = scrollBehavior,
-        navigationIcon = {
-            IconButton(onClick = onBackPressed) {
-                Icon(
-                    imageVector = Icons.AutoMirrored.Outlined.ArrowBack,
-                    contentDescription = LocalContext.current.getString(R.string.backButton),
-                    tint = MaterialTheme.colorScheme.onPrimary
-                )
+    var selectedFileIndex by remember { mutableIntStateOf(optionPosition) }
+    // TODO: Subject to change to use a tabbed view #4568
+    Column {
+        CodePagerBar(selectedFileIndex, codePagerTitles, onFileClicked = {
+            selectedFileIndex = it
+        })
+        if (codePagerTitles[selectedFileIndex].contains(".md")) {
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
+                verticalArrangement = Arrangement.Center
+            ) {
+                // TODO, should we render this screenshot URL? #4563
+                //val screenshotURL by remember { mutableStateOf("${sampleData?.screenshotURL}") }
```

Comment:
If it is not too much of a change, we could use the URL here to display the screenshot above the `ReadmeView`, or we can tackle this in #4751